packages/discord.js/src/structures/VoiceChannel.js

```diff
@@ -63,6 +63,10 @@ class VoiceChannel extends BaseGuildVoiceChannel {
     }
 
     if ('nsfw' in data) {
+      /**
+       * If the guild considers this channel NSFW
+       * @type {boolean}
+       */
```

Comment:
This is already in the constructor, and it shows in the documentation. I don't think this is necessary
https://github.com/discordjs/discord.js/blob/65d1879c0ae2d8d820323d0d835b1e5e3d071e57/packages/discord.js/src/structures/VoiceChannel.js#L23-L26

Reply:
I only added it because This change was recommended in #8136 and all other properties have types.

Reply:
`nsfw` has a type too, but it's being set in the constructor. All other properties are being set in the `_patch()` method, `nsfw` is an exception which is being set in both the constructor and the `_patch()` method